pkg/cmd/server/start/start_master.go

```diff
@@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
 		kc.RunGCController(gcClient)
 
+		kc.RunServiceLoadBalancerController()
```

Comment:
Have we taken into account how we'll change this code path to use versioned
clients?

On Tue, Apr 26, 2016 at 11:04 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_master.go
> <https://github.com/openshift/origin/pull/8633#discussion_r61102578>:
>
> > @@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
> >  		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
> >  		kc.RunGCController(gcClient)
> >
> > +		kc.RunServiceLoadBalancerController()
>
> Yeah, there's still a mix of the old and new implementations. Been trying
> to catch anything new going in.
>
> Agree.
>
> As a note for whoever does the rebase, we need to sort out what we do
> about shared cache ACL-ing. @liggitt <https://github.com/liggitt>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8633/files/a7a264ec9a036439b5959ec6c5d2d2234057ed81#r61102578>
>